app/test_split_integer.py

```diff
@@ -2,20 +2,30 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert (
+        sum(split_integer(100, 7)) == 100
+    ), "sum of parts must be equal to value"
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert (
+        split_integer(5, 5) == [1, 1, 1, 1, 1]
```

Comment:
this test and the message suggests that we are testing when values is equal to number of parts , but actually we are testing if result will contain equal parts if value is divisible without a remainder